server/project/parse.ts

```diff
@@ -77,3 +78,40 @@ export function parseRouterFunctions(code: string) {
 
   return routerFunctions;
 }
+
+export function createOperation(code: string) {
+  const routeRegExp =
+    /@Router\.(get|post)\("([^"]+)"\)\s+async\s+(\w+)\(((?:\s*\w+\s*:\s*\w+\s*,?\s*)+)\)\s*{/;
+  const match = routeRegExp.exec(code);
```

Comment:
Why not to use the function above? That one should also handle all comments and other HTTP methods.